scara_kinematics_node:
  link_0_height: 0.15
  link_1_length: 0.4
  link_2_length: 0.3